paparazzi/paparazzi-gradle-plugin/build.gradle

```diff
@@ -40,7 +40,7 @@ def generateVersion = tasks.register("pluginVersion") {
     versionFile.parentFile.mkdirs()
     versionFile.text = """// Generated file. Do not edit!
 package app.cash.paparazzi
-const val VERSION = "${project.version}"
+const val VERSION = "${project.property("VERSION_NAME")}"
```

Comment:
mind moving this instead to the paparazzi/build.gradle line 23 as:

```
version = property("VERSION_NAME") as String
```

keeps things closer to how they were, and prevents possible new modules from needing to remember this (if they need it).

Reply:
Done a2b31a4